src/language/scopes.ts

```diff
@@ -69,6 +69,12 @@ export class Scope {
       return this.props[name]
     }
 
```

Comment:
I think it would be better to add a flag to lookup that explicitly enables lookup in transcluded scopes. That way, the semantics of things like color lookup don't change.